Makefile.docker

```diff
@@ -0,0 +1,28 @@
+# Makefile.docker contains the shared tasks for building, tagging and pushing Docker images.
+# This file is included into the Makefile files which contain the Dockerfile files (E.g.
+# kafka-base, kafka etc.).
+#
+# The DOCKER_ORG (default is name of the current user) and DOCKER_TAG (based on Git Tag,
+# default latest) variables are used to name the Docker image. DOCKER_REGISTRY identifies
+# the registry where the image will be pushed (default is Docker Hub).
+TOPDIR=$(dir $(lastword $(MAKEFILE_LIST)))
+DOCKERFILE_DIR     ?= ./
+DOCKER_REGISTRY    ?= #REGISTRY
```

Comment:
is the "#" a typo?

Reply:
I wasn't sure what kind of default we should put here so I left it blank for the user to specify a registry of their choosing

Reply:
Should I make it more explicit (with a comment) that a user must supply a registry or should I populate it with "oc cluster up" default registry: 172.30.1.1:5000 ( This is what I change it to when I do local development)?